sys-apps/the_silver_searcher/the_silver_searcher-0.33.0.recipe

```diff
@@ -12,7 +12,7 @@ COPYRIGHT="2013-2016 Geoff Greer"
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/ggreer/the_silver_searcher/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="c5b208572e5cfc8a3cf366e8eb337b0c673c2ffa90c1ad90dfdcbe78251ba4cc"
+CHECKSUM_SHA256="351ab79ada811fd08f81296de10a7498ea3c46b681d73696d5a2911edbdc19db"
 
 ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
I've successfully built (and tested) this recipe on all architectures, including x86_64, so you might wish to change this line to:
```
ARCHITECTURES="!x86_gcc2 x86 x86_64"
```
While at it, could you also add this at the end of the recipe?
```

TEST()
{
	make check
}
```
This is a good idead because the generated Makefile does have a **`checkfs`** target.

Thanks!